applications/app/services/FacebookGraphApi.scala

```diff
@@ -16,7 +16,7 @@ case class URLResponse(id: String, share: ShareObject)
 case class ShareObject(share_count: Int)
 
 class FacebookGraphApiClient(wsClient: WSClient) extends implicits.WSRequests with ExecutionContexts {
-  val apiRootUrl = "https://graph.facebook.com/v2.8/"
+  val apiRootUrl = s"https://graph.facebook.com/v${Configuration.facebook.graphApi.version}/"
```

Comment:
Not really, but I like the idea of being able to bump the version without a code change